panther_driver/src/driver_node.py

```diff
@@ -138,6 +141,31 @@ def __init__(self, name) -> None:
         self._driver_state_msg.rear.right_motor.motor_joint_name = self._wheels_joints_names[3]
         self._driver_state_publisher = rospy.Publisher('driver/motor_controllers_state', DriverState, queue_size=1)
 
+        self._faults_list = [
```

Comment:
```suggestion
        self._driver_fault_flags = [
```

I don't think `list` is necessary. Also adding `driver` indicates where they relate